.github/ISSUE_TEMPLATE/operations.md

```
@@ -0,0 +1,129 @@
+## Technical Details
+
+**Component:**  
+<!-- Which service or infrastructure component? Backend API / Frontend / Database / Redis / Queue Workers / Contract / Load Balancer / etc. -->
+
+**Symptoms:**  
+<!-- Observable symptoms: error rates, latency, resource usage, etc. -->
+
+**Metrics/Logs:**  
+<!-- Paste relevant metrics, logs, or monitoring data -->
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)